[TECH] Migre Pix App vers `ember-mirage`.

HEYGUL opened the pull request:
## 🌸 Problème

<!-- Décrivez ici le besoin ou l'intention couvert par cette Pull Request. -->

## 🌳 Proposition

On suit le guide de migration disponible ici : https://github.com/bgantzler/ember-mirage/blob/main/docs/migration.md
<!-- Ajoutez à cet endroit, si nécessaire, des détails concernant la solution technique retenue et mise en oeuvre, des difficultés ou problèmes rencontrés. -->

## 🐝 Remarques

<!-- Des infos supplémentaires, trucs et astuces ? -->

## 🤧 Pour tester

<!-- Les instructions pour reproduire le problème, les profils de test, le parcours spécifique à utiliser, etc. -->

HEYGUL committed:
tech(mon-pix): declare all mirage models to prepare migration to ember-mirage

This commit aims to implement step #1 of `ember-mirage` migration guide.
See https://github.com/bgantzler/ember-mirage/blob/main/docs/migration.md
for further information.

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr12063.review.pix.fr)
  * [App (.org)](https://app-pr12063.review.pix.org)
  * [Orga (.fr)](https://orga-pr12063.review.pix.fr)
  * [Orga (.org)](https://orga-pr12063.review.pix.org)
  * [Certif (.fr)](https://certif-pr12063.review.pix.fr)
  * [Certif (.org)](https://certif-pr12063.review.pix.org)
  * [Junior](https://junior-pr12063.review.pix.fr)
  * [Admin](https://admin-pr12063.review.pix.fr)
  * [API](https://api-pr12063.review.pix.fr/api/)
  * [API MaDDo](https://pix-api-maddo-review-pr12063.osc-fr1.scalingo.io/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr12063.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr12063/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr12063/environment)
  * [scalingo api-maddo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-maddo-review-pr12063/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr12063/environment)

HEYGUL committed:
tech(mon-pix): disable serializer autodiscovery to prepare migration to ember-mirage

This commit aims to implement step #2 of `ember-mirage` migration guide.
See https://github.com/bgantzler/ember-mirage/blob/main/docs/migration.md
for further information.

HEYGUL committed:
tech(mon-pix): remove auto start of the server to prepare migration to ember-mirage

This commit aims to implement step #6 of `ember-mirage` migration guide.
See https://github.com/bgantzler/ember-mirage/blob/main/docs/migration.md
for further information.

HEYGUL committed:
tech(mon-pix): load explicitly factories, serializers and models to prepare migration to ember-mirage

    This commit aims to implement step #3 of `ember-mirage` migration guide.
│   See https://github.com/bgantzler/ember-mirage/blob/main/docs/migration.md

HEYGUL committed:
wip tech(mon-pix): migrate to ember-mirage

HEYGUL committed:
wip

HEYGUL commented:
Je ferme la PR car je n'ai pas envie d'y passer plus de temps.
